pandas/core/indexes/base.py

```diff
@@ -2529,15 +2529,20 @@ def set_value(self, arr, key, value):
     def _get_level_values(self, level):
         """
         Return an Index of values for requested level, equal to the length
-        of the index
+        of the index.
 
         Parameters
         ----------
-        level : int
+        level : int or level name
 
         Returns
         -------
-        values : Index
+        self : Index
```

Comment:
You aren't returning `self` are you?  I would leave this line alone and instead add a line below explaining what the output is.

Reply:
Actually it is returning self. this method probably meant to make the API the same for MultiIndex and other Index types, so for Index types this really doesn't do much. The name ``values`` is used for MultiIndex, but isn't used here...

should I still change it?

Reply:
@topper-123 : Sorry, I wasn't able to check the actual file on my phone.  Yes, you are correct that `self` is returned.  While I do stand corrected, I do think leaving it unchanged is preferable for consistency.  You can however, in a line beneath that explain that `values` in this implementation is just `self` because there is only one level.